qa/suites/rados/thrash-erasure-code/thrashers/bluestore-reshard.yaml

```diff
@@ -0,0 +1,5 @@
+tasks:
+- thrashosds:
+    timeout: 1200
+    chance_bluestore_reshard: 1
```

Comment:
I'd suggest adding these options to an existing thrash facet to avoid adding to the matrix, e.g. adding the bluestore options to thrashers/default.yaml

Reply:
should add these to at lest the octopus-x upgrade suites as well, e.g. in qa/suites/upgrade/octopus-x/stress-split-no-cephadm/3-thrash/default.yaml and qa/suites/upgrade/octopus-x/stress-split/1-start.yaml